common/src/service/HttpClientHelper.cs

```diff
@@ -919,5 +894,37 @@ public void Dispose()
             Task<Exception> exception = mapToExceptionFunc(response);
             return await exception.ConfigureAwait(false);
         }
+
+        internal static HttpClient CreateDefaultClient(IWebProxy webProxy, Uri baseUri, int connectionLeaseTimeoutMilliseconds)
+        {
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+#if NETCOREAPP
+            SocketsHttpHandler httpMessageHandler = new SocketsHttpHandler();
+#else
+            HttpClientHandler httpMessageHandler = new HttpClientHandler();
+#endif
+#pragma warning restore CA2000 // Dispose objects before losing scope
+
+#if !NET451 && !NETCOREAPP
+            httpMessageHandler.SslProtocols = TlsVersions.Instance.Preferred;
+            httpMessageHandler.CheckCertificateRevocationList = TlsVersions.Instance.CertificateRevocationCheck;
```

Comment:
this doesn't work for netcoreapp?

Reply:
it looks like you can use this for `SocketsHttpHandler`: https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslclientauthenticationoptions?view=netstandard-2.1

Reply:
SocketsHttpHandler (which is the class that httpMessageHandler is in the netcoreapp case) doesn't have a SslProtocols field or a CheckCertificateRevocationList field

Reply:
you can set ssloptions here: https://docs.microsoft.com/en-us/dotnet/api/system.net.http.socketshttphandler.ssloptions?view=net-5.0#System_Net_Http_SocketsHttpHandler_SslOptions. Does this not work?

Reply:
Good catch, I can add that